x/stakeibc/keeper/host_zone.go

```diff
@@ -215,14 +227,19 @@ func (k Keeper) AddValidatorToHostZone(ctx sdk.Context, chainId string, validato
 		valWeight = minWeight
 	}
 
+	// Determine the slash query checkpoint for LSM liquid stakes
+	params := k.GetParams(ctx)
+	queryThreshold := sdk.NewDecWithPrec(int64(params.ValidatorSlashQueryThreshold), 2) // percentage
+	checkpoint := queryThreshold.Mul(sdk.NewDecFromInt(hostZone.TotalDelegations)).TruncateInt()
```

Comment:
This is the initialization only for a new validator... make sense just confirming.